scripts/build_torch_wheels.sh

```diff
@@ -67,7 +67,8 @@ function build_and_install_torch() {
   # Only checkout dependencies once PT commit ID checked out.
   git submodule update --init --recursive
   # Apply patches to PT which are required by the XLA support.
-  xla/scripts/apply_patches.sh
+  # xla/scripts/apply_patches.sh
+  $(echo $0 | sed s/build_torch_wheels/apply_patches/)
```

Comment:
@dlibenzi @jysohn23 this line is equivalent to line 70 when run from the pytorch directory, but it doesn't fail when run from somewhere else (which I incidentally did today).